components/HomeSection/Assets.tsx

```diff
@@ -65,13 +63,13 @@ const AssetsHomeSection: FC<Props> = ({ date }) => {
       }
       return randomTokens
     }
-    return defaultAssetData?.assets?.nodes.map((x) => x.id)
-  }, [defaultAssetData, date])
+    return (defaultAssetData?.assets?.nodes || []).map((x) => x.id)
```

Comment:
Revert, otherwise it will not be `undefined`, because of that `skip` will not work.
```suggestion
    return defaultAssetData?.assets?.nodes.map((x) => x.id)
```

Reply:
Good catch: https://github.com/liteflow-labs/starter-kit/pull/383/commits/d61c9b39d3d0dec3eabbd0d82441d4f021359a23